modules/log/log.go

```diff
@@ -180,6 +203,9 @@ func Close() {
 // Log a message with defined skip and at logging level
 // A skip of 0 refers to the caller of this command
 func Log(skip int, level Level, format string, v ...interface{}) {
+	mu.RLock()
+	defer mu.RUnlock()
+
 	l, ok := NamedLoggers[DEFAULT]
```

Comment:
I think we only needs a lock about the `NamedLoggers` I don't think `l.Log(skip+1, level, format, v...)`  needs a lock.

Reply:
Done.